External/FEXCore/Source/Utils/MemberFunctionToPointer.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <FEXCore/Utils/LogManager.h>
+
+#include <cstdint>
+
+namespace FEXCore::Utils {
+
+/**
+ * @brief Casts a class's member function pointer to a raw pointer that we can JIT
+ *
+ * Has additional validation to ensure we aren't casting a class member that is invalid
+ */
+template <typename PointerToMemberType>
+class MemberFunctionToPointerCast final {
+  public:
+    MemberFunctionToPointerCast(PointerToMemberType Function) {
+      union PointerCastType {
```

Comment:
The union isn't needed anymore now - in fact, dropping it will avoid the somewhat puzzling memcpy of 16 bytes through the 8-byte RawPointer member :)

Looks good other than that, thanks!

Reply:
Ah, you're right.
Resolved now.